synadm/api.py

```diff
@@ -110,33 +114,34 @@ def user_modify(self, user_id, password, display_name, threepid,
         return self.query("put", f"v2/users/{user_id}", data=data)
 
     def room_list(self, _from, limit, name, order_by, reverse):
-        """ List and search rooms
-        """
-        return self.query("get", "v1/rooms", params={
-            "from": _from,
-            "limit": limit,
-            "search_term": name,
-            "order_by": order_by,
-            "dir": "b" if reverse else None
-        })
+        """List and search rooms"""
+        return self.query(
+            "get",
+            "v1/rooms",
+            params={
+                "from": _from,
+                "limit": limit,
+                "search_term": name,
+                "order_by": order_by,
+                "dir": "b" if reverse else None,
+            },
+        )
 
     def room_details(self, room_id):
-        """ Get details about a room
-        """
+        """Get details about a room"""
         return self.query("get", f"v1/rooms/{room_id}")
 
     def room_members(self, room_id):
-        """ Get a list of room members
-        """
+        """Get a list of room members"""
         return self.query("get", f"v1/rooms/{room_id}/members")
 
-    def room_delete(self, room_id, new_room_user_id, room_name, message,
-                    block, no_purge):
-        """ Delete a room and purge it if requested
-        """
+    def room_delete(
+        self, room_id, new_room_user_id, room_name, message, block, no_purge
+    ):
```

Comment:
For example, this is awful IMHO, the version on the left is properly readable already and spare a line. Using a magic auto-formatting tool is not always the best solution. Still: IMHO, Thanks for the effort anyway :-) I currently like to use recommendations provided by a tool called flake8.